test/adapter.test.ts

```diff
@@ -757,7 +760,7 @@ describe('query adapter', () => {
               status: {category: 'c1a', detail: 'd1a'}
             },
             {
-              changedAt: 1667871145261,
+              changedAt: '2022-11-08T01:32:25.261Z',
```

Comment:
Test a mix of epoch millis and ISO timestamps inside embedded object lists are converted to epoch millis.